.nycrc

```diff
@@ -0,0 +1,8 @@
+{
+  "branches": 87,
```

Comment:
What does this imply? Can you please explain.

Reply:
Currently, not all `else` statements are tested in `index.js`, we have https://github.com/oppia/oppiabot/blob/d70b6515128b26ac5a06954e1653c7c5626d5a69/index.js#L26 but we are not testing the else case.

Reply:
The branches represent the untested else statement.

Reply:
Ok, got it. But what if we add new if statements and the score again changes - we do not need that anyway, so we should remove branches from the config. WDYT?

Reply:
The branches help in testing the actual implementation (the modules).

Reply:
The value in the config sets the minimum that the branch should ever go and I think it's safe?

Reply:
Ok, sg then. Thanks for the explanation.